easybuild/easyconfigs/a/assembly-stats/assembly-stats-1.0.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'assembly-stats'
+version = '1.0.1'
+
+homepage = 'https://github.com/sanger-pathogens/assembly-stats'
+description = 'Get assembly statistics from FASTA and FASTQ files.'
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/sanger-pathogens/assembly-stats/releases']
+sources = [{
+    'git_config': {
```

Comment:
Consider the git_config more or a "last resort"
We can't reliably checksum it (yet) and we prefer to take the release tarballs, i.e:
`https://github.com/sanger-pathogens/assembly-stats/archive/refs/tags/v1.0.1.tar.gz` in this case.

```python
github_account = 'sanger-pathogens'
source_urls = [GITHUB_SOURCE]
sources = ['v%(version)s.tar.gz']
```
should work here i believe